Dockerfile

```diff
@@ -1,23 +1,50 @@
 FROM python:3.12-slim AS base
 
-RUN apt-get update && apt-get install -y \
-    build-essential \
-    libpq-dev \
-    ffmpeg \
-    && rm -rf /var/lib/apt/lists/*
+ARG UID=1000
+ARG GID=1000
 
-ENV PYTHONUNBUFFERED 1
+RUN apt-get update && apt-get install -y --no-install-recommends \
+      ffmpeg \
+    && apt-get clean all \
+    && rm -rvf /var/lib/apt/* /var/cache/apt/* \
+    && find / -type d -name "__pycache__" -exec rm -rv {} + 2>/dev/null || true
+
+ENV VIRTUAL_ENV="/app_venv"
+ENV PATH="${VIRTUAL_ENV}/bin:${PATH}"
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+
+RUN \
+    groupadd -fg $GID app \
```

Comment:
Are we expected to set `GID` and `UID` in our environment variables (.env)? If yes, can we update the `local.example.py`
<img width="1536" alt="Screenshot 2025-02-12 at 8 46 50 PM" src="https://github.com/user-attachments/assets/4b11a7da-0a78-4d28-b903-0ab45904ea7e" />